DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/basics/theDocumentStore/metadata.json

```diff
@@ -0,0 +1,211 @@
+                "",
+                "  * Open a Session for database transactions",
+                "  * Create & deploy indexes",
+                "  * Bulk documents actions",
+                "  * Manage the server with low-level operations commands",
+                "  * Changes API - receive messages from the server",
+                "  * Perform custom actions in response to Session's events",
+                "  * Subscription tasks management",
+                "  * Export & Import database data."
+            ],
+            "Assets": [
+                {
+                    "Title": "The Document Store",
+                    "Type" : "link",
+                    "Url"  : "https://github.com/ravendb/book/blob/v4.0/Ch02/Ch02.md#the-document-store"
+                },
+                {
+                    "Title": "What is a Document Store",
+                    "Type" : "link",
+                    "Url"  : "https://ravendb.net/docs/article-page/4.1/csharp/client-api/what-is-a-document-store"
+                },
+                {
+                    "Title": "Creating a document store",
+                    "Type" : "link",
+                    "Url"  : "https://ravendb.net/docs/article-page/4.2/java/client-api/creating-document-store"
+                },
+                {
+                    "Title": "Sample program",
+                    "Type" : "link",
+                    "Url"  : "https://github.com/ravendb/ravendb-jvm-tutorials#document-store"
+                }
+            ]
+        },
+        {
+            "Title": "The private store variable",
+            "Slug" : "step-2",
+            "DescriptionLines": [
+                "* The private static variable `store` will hold a reference to the single Document Store instance that will be created.",
+                "  The public _getStore()_ method (see step 6) will be used to access this instance."
+            ],
+            "Assets": [
+                {
+                    "Title": "Singleton Pattern",
+                    "Type" : "link",
+                    "Url"  : "https://www.journaldev.com/1377/java-singleton-design-pattern-best-practices-examples"
+                }
+            ]
+        },
+        {
+            "Title": "Create the document store configuration",
+            "Slug" : "step-3",
+            "DescriptionLines": [
+                "* Method createDocumentStore() returns a configured `DocumentStore` object that is used to initialize the _store_ variable in the previous step."
+            ],
+            "Assets": []
+        },
+        {
+            "Title": "Configure the DocumentStore Object",
+            "Slug" : "step-4",
+            "DescriptionLines": [
+                "* At this point, before initializing the Document Store (in step 5), the following info can be configured:",
+                "",
+                "  * _URLs_ - Specify a list of URL addresses pointing to the RavenDB server nodes composing your cluster.",
```

Comment:
put those props lower case - it will be more java like style.

Reply:
or write: getDatabase/setDatabase

Reply:
done